rental.rb

```diff
@@ -2,12 +2,12 @@ class Rental
   attr_accessor :date, :person, :book
 
   def initialize(date, book, person)
-    @date =date
+    @date = date
 
     @book = book
-    #book.rentals << self
+    # book.rentals << self
 
     @person = person
-   # person.rentals << self
+    # person.rentals << self
```

Comment:
[Optional Changes]
These changes are not major but highly recommend you consider fixing them

- [ ] Kindly remove all unused commented codes in your codebase.